portal-api/src/external-api/bank-holidays.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
+const dotenv = require('dotenv');
+
+dotenv.config();
+const { EXTERNAL_API_URL, EXTERNAL_API_KEY } = process.env;
+
+const headers = {
+  'Content-Type': 'application/json',
+  'x-api-key': EXTERNAL_API_KEY,
+};
+
+/**
+ * Fetches details of UK bank holidays from external-api.
+ * @returns {Promise<object>}
+ */
+const getBankHolidays = async () => {
+  const response = await axios.get(`${EXTERNAL_API_URL}/bank-holidays`, { headers });
+  return response.data;
+};
+
+/**
+ * Fetches details of UK bank holidays for the specified region from external-api.
```

Comment:
In my head we only cared about england as UKEF makes the MGA and they are based in england.

Could I get you to double check this with the team please?

Reply:
Do we just delete this? or is this something francesca made we need to delete?

Reply:
I did this but it's going to clash with @francescastocco's changes [here](https://github.com/UK-Export-Finance/dtfs2/pull/2197/files#diff-ed25e4c474fb57f17d8bb0eb959cca28859df0636671df0358e4db5ace04f908) when I come to merge.

I'm not sure I understand the business well enough to know whether we'll ever need any holidays other than for England, so happy to leave that call up to you